src/main/java/wolox/training/repositories/BookRepository.java

```diff
@@ -0,0 +1,8 @@
+package wolox.training.repositories;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import wolox.training.models.Book;
+
+public interface BookRepository extends JpaRepository<Book, Long> {
+    Book findByAuthor(String author);
```

Comment:
This method should return an Optional<Book> to avoid potentially returning a null value :) 
Here's some info if you need it! https://docs.oracle.com/javase/8/docs/api/java/util/Optional.html